runhouse/rns/login.py

```diff
@@ -105,22 +105,29 @@ def login(
 
         # Default ssh secret
         if not rns_client.default_ssh_key:
+            from runhouse import provider_secret
+
             default_ssh_path = typer.prompt(
-                "Input the private key path for your default SSH key to use for launching clusters (e.g. ~/.ssh/id_rsa), or press `Enter` to skip",
+                "Input the private key path for your default SSH key to use for "
+                "launching clusters (e.g. ~/.ssh/id_rsa), or press `Enter` to automatically create a new key pair:",
                 default="",
             )
 
-            if default_ssh_path:
-                from runhouse import provider_secret
+            if not default_ssh_path:
```

Comment:
I still think having a default ssh path should be optional and we shouldn't explicitly force it in oss, and that generating the sky key if there's no default + doesn't exist should happen at the time of the first sky launch rather than at login time

(maybe the user will only use static clusters, or wants to load a key elsewhere or set it later on)